_layouts/media.html

```diff
@@ -105,6 +105,7 @@
 
 var talks = [
 	{ id: "wSpPjUPIEVU", title: "OSDC 2015 Foreman in your data center", author: "Stephen Benjamin" },
+	{ id: "b18Us1nAKMA", title: "PuppetCamp 2015: Selling management and developers on Puppet", author: "David Patterson"},
```

Comment:
Wow.  I don't even know how that happened.  Yes, you're correct, the ID should be Qo3Bkm7dqlU.

Being new at this, do I need to create a new pull request for that?  The embarrassment is palpable over here, heh.